yt_dlp/extractor/facebook.py

```diff
@@ -616,6 +625,32 @@ def parse_graphql_video(video):
                                 'url': playable_url,
                             })
                     extract_dash_manifest(video, formats)
+
+                    automatic_captions, subtitles = {}, {}
```

Comment:
why was this moved?

Reply:
to avoid complicated traverse_obj in
'''
460            subs_data = traverse_obj(post, (..., 'video', ..., 'attachments', ..., lambda k, v: (
461                k == 'media' and str(v['id']) == video_id and v['__typename'] == 'Video')))
'''
because events use different keys.
Moving subtitle extraction inside 'parse_graphql_video()' then we can remove these two lines

Reply:
Okay that makes sense. But moving the whole code block made the diff much less obvious, and when there were 3 other unrelated changes in 1 PR I didn't even know what to look for. So thank you for separating the PRs